binder/requirements.txt

```diff
@@ -4,6 +4,7 @@ ipywidgets~=8.0.7
 ipympl~=0.9.3
 pandas~=2.0.3
 altair~=5.0.1
+rich==12.6.0
```

Comment:
```
python -m pip index versions rich
```

shows the latest is `13.5.2`. Is there a reason to keep this pinned to `12.6`? If not, might as well bump it.

Reply:
it's fine to bump it up. I just picked 12.6 since i already had that version.

Reply:
Thanks. I bumped it in the commit just before the reverting of the language edits you pointed out. Resolving this now.